frontend/src/components/Table/Table.scss

```diff
@@ -82,7 +82,7 @@
       width: 100%;
       height: 100%;
       align-items: center;
-      justify-content: space-evenly;
+      justify-content: flex-start;
```

Comment:
we have some pretty inconsistent table cell justification across the app, would love to clean that all up sometime.

Reply:
not actually a problem in your PR btw, just thinking out loud.